app/views/apps/_service_notification_fields.html.haml

```diff
@@ -16,6 +16,7 @@
         %div.notification_params{:class => (w.object.is_a?(notification_service) ? 'chosen ' : '') << notification_service.label}
           - notification_service::FIELDS.each do |field, field_info|
             = w.label field, field_info[:label] || field.to_s.titleize
+            %em= field_info[:hint]
```

Comment:
I guess an if statement here wouldn't hurt. That said, if the `hint` is blank the `em` will have 0 `height`.